lib/nifty_services/base_crud_service.rb

```diff
@@ -12,22 +11,25 @@ def record_type(record_type, options = {})
           record_type
         end
 
-        record_alias = options.delete(:alias_name) || record_type.to_s.underscore
+        record_alias = options.delete(:alias_name)
```

Comment:
@viniciuszago That's why `||` exists here, I mean, there's no point changing this, what do you thnk?